pkgs/applications/window-managers/i3/default.nix

```diff
@@ -24,11 +24,12 @@ stdenv.mkDerivation rec {
 
   doCheck = stdenv.system == "x86_64-linux";
 
-  checkPhase = ''
+  checkPhase = if stdenv.system == "x86_64-linux" then
```

Comment:
This will read a bit better as `stdenv.lib.optionalString (stdenv.system == "x86_64-linux") ''...''`.